src/current/_includes/common/upgrade/prepare-to-upgrade-cloud.md

```diff
@@ -0,0 +1,12 @@
+Before beginning a major-version upgrade:
+
+1. **CockroachDB {{ site.data.products.advanced }}**: Verify the overall health of your cluster using the [DB Console]({% link cockroachcloud/tools-page.md %}#access-the-db-console.md):
+    - Under **Node Status**, make sure all nodes that should be live are listed as such. If any nodes are unexpectedly listed as `SUSPECT` or `DEAD`, identify why the nodes are offline and either restart them or [decommission]({% link {{ site.current_cloud_version }}/node-shutdown.md %}?filters=decommission#remove-nodes) them before beginning your upgrade. If there are `DEAD` and non-decommissioned nodes in your cluster, the upgrade cannot be finalized.
```

Comment:
@joshimhoff can you comment on whether this is something cloud customers need to do? Does our automation ever allow suspect or dead nodes to stick around?

Reply:
Confirming what @joshimhoff said!

Reply:
Done! (Joel confirmed offline)